docs/coroutine-context-and-dispatchers.md

```diff
@@ -155,8 +155,36 @@ The unconfined dispatcher should not be used in general code.
 
 Coroutines can suspend on one thread and resume on another thread. 
 Even with a single-threaded dispatcher it might be hard to
-figure out what the coroutine was doing, where, and when. The common approach to debugging applications with 
-threads is to print the thread name in the log file on each log statement. This feature is universally supported
+figure out what the coroutine was doing, where, and when if you don't have special tooling. 
+
+#### Debugging with special tooling 
+
+The Coroutine Debugger of the Kotlin plugin simplifies debugging coroutines.
```

Comment:
It's worth to mention that it works only in Intellij IDEA

Reply:
✅